src/target/riscv/riscv-013.c

```diff
@@ -913,10 +918,21 @@ static int register_read_abstract(struct target *target, uint64_t *value,
 	return ERROR_OK;
 }
 
-static int register_write_abstract(struct target *target, uint32_t number,
-		uint64_t value, unsigned size)
+static unsigned int register_size(struct target *target, enum gdb_regno number);
```

Comment:
Please, move this forward declaration to the top of the file - to the same place where the other forward declarations are.

Reply:
Addressed.